docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -620,16 +613,56 @@ DOM node or the timeout exceeds.
 Note that you can additionally require that the node returned by the selector is visible.
 To do this, use the [visibilityCheck](selector-options.md#optionsvisibilitycheck) option.
 
+## Adding Custom Properties to Element State
+
+TestCafe allows you to extend [element state](#obtain-element-state) with custom properties calculated on the client side.
+
+To do this, use the selector's `addCustomDOMProperties` method.
+
+```text
```

Comment:
I don't think so. I don't know why but every where we use `text` in such cases:
[first](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/test-api/selecting-page-elements/selectors.md#creating-selectors), [second](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/test-api/assertions/index.md#assertion-structure)

For example in the second link:
text:
```text
await t.expect( actual ).eql( expected, message, options );
```
and than an example with `js`